ci-scripts/e2e-cypress.sh

```diff
@@ -25,6 +26,11 @@ do
             shift
             shift
             ;;
+        '--env' )
```

Comment:
need to add --environment as well as --env to match line 12

Reply:
done